kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/ProductOrderService.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.good.patterns.challenges;
+
+public class ProductOrderService
+{
+    private InformationService informationService;
+    private OrderService orderService;
+    private OrderRepository orderRepository;
```

Comment:
tutaj pola nie są final w przeciwieństwie do pozostałych klas - jakiś konkretny powód ?

Reply:
Uznałem, że klasa ta nie jest klasą elementarną i nie będzie to potrzebne ale bez przekonania - czyli działam trochę po omacku.